scss/inc/_tooltip.scss

```diff
@@ -22,6 +22,7 @@ $dark : #404040;
 .tooltip[role="tooltip"] {
   position: absolute;
   /*background: #FFC107;*/
+  writing-mode: horizontal-tb; /* vertical-rl sizing & anchoring not implemented */
```

Comment:
"Anchoring not implemented" - perhaps this is explaining that the issues I posted [here](https://github.com/oat-sa/extension-tao-itemqti/pull/2671#pullrequestreview-2595730833) are kind of expected?

Reply:
No, it's not related. 
Also, for TextEntryInteraction I did implement `vertical-rl` for tooltip (it was just css change), will need to merge here...

Reply:
https://github.com/oat-sa/tao-item-runner-qti-fe/pull/419/commits/6d2ac65bfcc273b5b5d7a0dc3a4b0d55fc18a6f0 & https://github.com/oat-sa/tao-core-ui-fe/pull/630/commits/b36d17b4711fb7d9c22ed82e7fe1c3e950c890a0